packages/@aws-cdk/aws-fsx/lib/lustre-file-system.ts

```diff
@@ -0,0 +1,305 @@
+import {Connections, ISubnet, Peer, Port, SecurityGroup} from '@aws-cdk/aws-ec2';
+import {Construct} from '@aws-cdk/core';
+import {FsxFileSystemAttributes, FsxFileSystemBase, FsxFileSystemProps, IFsxFileSystem} from './file-system';
+import {CfnFileSystem} from './fsx.generated';
+
+/**
+ * The different kinds of file system deployments used by Lustre.
+ */
+export enum FsxLustreDeploymentType {
+  /**
+   * Original type for shorter term data processing. Data is not replicated and does not persist on server fail.
+   */
+  SCRATCH_1 = 'SCRATCH_1',
+  /**
+   * Newer type for shorter term data processing. Data is not replicated and does not persist on server fail.
+   * Provides better support for spiky workloads.
+   */
+  SCRATCH_2 = 'SCRATCH_2',
+  /**
+   * Long term storage. Data is replicated and file servers are replaced if they fail.
+   */
+  PERSISTENT_1 = 'PERSISTENT_1'
+}
+
+/**
+ * The configuration for the Amazon FSx for Lustre file system.
+ *
+ * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-fsx-filesystem-lustreconfiguration.html
+ */
+export interface LustreConfiguration {
+  /**
+   * The type of backing file system deployment used by FSx.
+   */
+  readonly deploymentType: FsxLustreDeploymentType;
+
+  /**
+   * The path in Amazon S3 where the root of your Amazon FSx file system is exported. The path must use the same
+   * Amazon S3 bucket as specified in ImportPath. If you only specify a bucket name, such as s3://import-bucket, you
+   * get a 1:1 mapping of file system objects to S3 bucket objects. This mapping means that the input data in S3 is
+   * overwritten on export. If you provide a custom prefix in the export path, such as
+   * s3://import-bucket/[custom-optional-prefix], Amazon FSx exports the contents of your file system to that export
+   * prefix in the Amazon S3 bucket.
+   *
+   * @default s3://import-bucket/FSxLustre[creation-timestamp]
+   */
+  readonly exportPath?: string;
+
+  /**
+   * For files imported from a data repository, this value determines the stripe count and maximum amount of data per
+   * file (in MiB) stored on a single physical disk. Allowed values are between 1 and 512,000.
+   *
+   * @default 1024
+   */
+  readonly importedFileChunkSize?: number;
```

Comment:
Same note about units, work it into the property name for maximum clarity.

Reply:
Fixed.